workflows/readsimulator/main.nf

```diff
@@ -72,44 +70,21 @@ def multiqc_report = []
 
 workflow READSIMULATOR {
 
+    take:
+    ch_input
+
+    main:
     ch_versions        = Channel.empty()
-    ch_input           = Channel.fromSamplesheet("input")
     ch_simulated_reads = Channel.empty()
-    ch_taxids          = Channel.empty()
-    ch_accessions      = Channel.empty()
 
     if ( params.fasta ) {
         ch_fasta = Channel.fromPath(params.fasta)
     } else {
-        if ( params.ncbidownload_accessions ) {
-            ch_accessions = Channel.fromPath(params.ncbidownload_accessions)
-        } else if ( params.ncbidownload_taxids ) {
-            ch_taxids = Channel.fromPath(params.ncbidownload_taxids)
-        }
-
-        //
-        // MODULE: Download reference fasta files
-        //
-        NCBIGENOMEDOWNLOAD (
-            [ id:"ncbigenomedownload" ],
-            ch_accessions.ifEmpty([]),
-            ch_taxids.ifEmpty([]),
-            params.ncbidownload_group
-        )
-
```

Comment:
Are we putting NCBIGENOMEDOWNLOAD() somewhere else like a subworkflow?

Reply:
I think this must have been a merging error on my part. I will add it back in